packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -117,6 +117,12 @@
           "when": "view == salesforce-lightning-components && viewItem == external",
           "group": "inline"
         }
+      ],
+      "commandPalette": [
+        {
+          "command": "salesforce-lightning-quickopen",
+          "when": "sfdx:project_opened"
```

Comment:
I tested this and setting this causes the quick open command to no longer appear for the internal development scenario.